src/executables/utility.h

```diff
@@ -919,6 +919,8 @@ typedef struct _ha_config
 
 #define CHECK_SERVER              "Server"
 #define CHECK_HA_SERVER           "HA-Server"
+#define CHECK_HA_PAGE_SERVER      "HA-Page-Server"
+#define CHECK_HA_TRAN_SERVER      "HA-Tran-Server"
```

Comment:
Nit:

make consistency between CHECK_HA_TRAN_SERVER and CHECK_TRANSACTION_SERVER.
1. CHECK_HA_TRAN_SERVER -> CHECK_HA_TRANSACTION_SERVER
2. or viceversa

Reply:
OK..!